core/integration/features/liquidity-provision/0044-LIME-102.feature

```diff
@@ -0,0 +1,141 @@
+      | ETH/MAR22 | USD        | USD   | lqm-params           | log-normal-risk-model | margin-calculator-1 | 2                | fees-config-1 | price-monitoring | default-eth-for-future | 1e0                    | 0                         | SLA-22     |
+      | ETH/MAR23 | USD        | USD   | lqm-params           | log-normal-risk-model | margin-calculator-1 | 2                | fees-config-1 | price-monitoring | default-eth-for-future | 1e0                    | 0                         | SLA-23     |
+
+    And the following network parameters are set:
+      | name                                                | value |
+      | market.liquidity.bondPenaltyParameter               | 0.2   |
+      | market.liquidity.stakeToCcyVolume                   | 1     |
+      | market.liquidity.successorLaunchWindowLength        | 1h    |
+      | market.liquidity.sla.nonPerformanceBondPenaltySlope | 0.1   |
+      | market.liquidity.sla.nonPerformanceBondPenaltyMax   | 0.6   |
+      | validators.epoch.length                             | 10s   |
+      | market.liquidity.earlyExitPenalty                   | 0.25  |
+      | market.liquidity.maximumLiquidityFeeFactorLevel     | 0.25  |
+
+    Given the average block duration is "1"
+  @Now
+  Scenario: during liquidity auciton, in terms of the liquidity they are expected to supply: this only takes effect from the start of the next epoch 0044-LIME-102
+    Given the parties deposit on asset's general account the following amount:
+      | party  | asset | amount |
+      | lp1    | USD   | 100000 |
+      | lp2    | USD   | 100000 |
+      | lp3    | USD   | 100000 |
+      | party1 | USD   | 100000 |
+      | party2 | USD   | 100000 |
+      | party3 | USD   | 100000 |
+
+    And the parties submit the following liquidity provision:
+      | id   | party | market id | commitment amount | fee   | lp type    |
+      | lp_1 | lp1   | ETH/MAR22 | 6000              | 0.02  | submission |
+      | lp_2 | lp2   | ETH/MAR22 | 4000              | 0.015 | submission |
+
+    When the network moves ahead "4" blocks
+    And the current epoch is "0"
+    Then the parties place the following orders:
+      | party  | market id | side | volume | price | resulting trades | type       | tif     | reference |
+      | party1 | ETH/MAR22 | buy  | 10     | 900   | 0                | TYPE_LIMIT | TIF_GTC |           |
+      | party1 | ETH/MAR22 | buy  | 1      | 1000  | 0                | TYPE_LIMIT | TIF_GTC |           |
+      | party2 | ETH/MAR22 | sell | 10     | 1100  | 0                | TYPE_LIMIT | TIF_GTC |           |
+      | party2 | ETH/MAR22 | sell | 1      | 1000  | 0                | TYPE_LIMIT | TIF_GTC |           |
+
+    Then the opening auction period ends for market "ETH/MAR22"
```

Comment:
Considering we have two markets, and are moving forwards by blocks and epochs here, it's worth noting that this step may be a bit misleading. Both markets have the same opening auction duration, so we're effectively ending the auction period for both here (in fact, because we've moved forwards 4 blocks after the LP submission, we probably only need to move the network forwards by 1 block, or submit the orders with ticks